deploy/ansible/roles/iroha-cluster-deploy-node/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+  # docker container postgres settings
+  postgresName: tmp_postgres
+  postgresPort: 5432
+  postgresUser: psql
+  postgresPassword: psql
+  iroha_net: iroha_network  # name of docker network
+  containerConfPath: /opt/iroha_data  # path to folder with config files inside docker container
+
+  irohaDockerImage: hyperledger/iroha-docker # docker image name
```

Comment:
Must soon be switched to hyperledger/iroha:x86_64-develop-latest

Reply:
@bakhtin then, it will by the easiest PR.